docs/modules/data-connections/pages/data-connections-configuration.adoc

```diff
@@ -381,6 +381,147 @@ OPTIONS (
+<2> Hazelcast client configuration to connect to a remote cluster. See xref:clients:java.adoc#configuring-java-client[Configuring Java Client].
+You can specify an external file with the `client_xml_path` property instead of using an embedded configuration file.
+--
+
+YAML::
++
+--
+[source,yaml]
+----
+hazelcast:
+  data-connection:
+    my-remote-hazelcast:
+      type: Hz
+      properties:
+        client_yml: | <1>
+          hazelcast-client: <2>
+            cluster-name: dev
+            network:
+              cluster-members:
+                - 172.17.0.2:5701
+      shared: true
+----
+<1> (Required) Specify exactly one of `client_xml`, `client_yml`, `client_xml_path`, `client_yml_path`.
+<2> Hazelcast client configuration to connect to a remote cluster. See xref:clients:java.adoc#configuring-java-client[Configuring Java Client].
+You can specify an external file with the `client_yml_path` property instead of using an embedded configuration file.
+--
+
+Java::
++
+--
+[source,java]
+----
+config
+  .addDataConnectionConfig(
+    new DataConnectionConfig()
+      .setName("my-remote-hazelcast")
+      .setType("Hz")
+      .setProperty(
+              "client_xml", <1>
+              "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + <2>
+              "<hazelcast-client xmlns=\"http://www.hazelcast.com/schema/client-config\"" +
+              "      xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"" +
+              "      xsi:schemaLocation=\"http://www.hazelcast.com/schema/client-config" +
+              "      http://www.hazelcast.com/schema/client-config/hazelcast-client-config-5.3.xsd\">" +
+              "    " +
+              "    <cluster-name>dev</cluster-name>" +
+              "    <network>" +
+              "        <cluster-members>" +
+              "            <address>172.17.0.2:5701</address>" +
+              "        </cluster-members>" +
+              "    </network>" +
+              "</hazelcast-client>")
+      .setShared(true)
+  );
+----
+<1> (Required) Specify exactly one of `client_xml`, `client_yml`, `client_xml_path`, `client_yml_path`.
+<2> Hazelcast client configuration to connect to a remote cluster. See xref:clients:java.adoc#configuring-java-client[Configuring Java Client].
+You can specify an external file with the `client_xml_path` or `client_yml_path` property instead of using an embedded configuration file.
+--
+SQL::
++
+--
+Data connections created in SQL behave differently to those defined in members' configuration files or in Java.
+
+- To retain SQL-defined data connections after a cluster restart, you must enable xref:storage:configuring-persistence.adoc#sql[SQL metadata persistence]. This feature is available in the Enterprise Edition.
+- You can create or drop a data connection using SQL commands. To update a data connection, you need to drop and then recreate it.
+
+NOTE: While Hazelcast data connection can be created using SQL command it cannot yet be used from SQL, because there
```

Comment:
I don't think we need this note as the same text already appears the intro text before the tabbed examples. Here's a preview of the page https://deploy-preview-742--hardcore-allen-f5257d.netlify.app/hazelcast/5.4-snapshot/data-connections/data-connections-configuration#Hazelcast